website/content/docs/connect/config-entries/jwt-provider.mdx

```diff
@@ -8,6 +8,8 @@ description: |-
 
 This page provides reference information for the JWT provider configuration entry, which configures Consul to use a JSON Web Token (JWT) and JSON Web Key Set (JWKS) in order to add JWT validation to proxies in the service mesh. Refer to [Use JWT authorization with service intentions](/consul/docs/connect/intentions/jwt-authorization) for more information.
 
+~> **Note**: Consul must be configured to use ACLs with a default deny policy for JWT authorization with service intentions to work
```

Comment:
```suggestion
```
This is more of a usage content detail and should appear in the Requirements of any usage pages that involve using this configuration entry.